govcd/api_vcd_test.go

```diff
@@ -1426,3 +1446,39 @@ func (vcd *TestVCD) Test_NewRequestWitNotEncodedParamsWithApiVersion(check *C) {
 
 	fmt.Printf("Test: %s run with api Version: %s\n", check.TestName(), apiVersion)
 }
+
+func setBoolFlag(variable *bool, name, envVar, help string) {
+	if envVar != "" && os.Getenv(envVar) != "" {
+		*variable = true
+		return
+	}
+	flag.BoolVar(variable, name, *variable, help)
```

Comment:
This is actually tricky to read, because `variable` is used twice in the same line. And it's not clear how can it have the default value at this early point in time too. To clarify, please add a comment about this line and also about this function.

Reply:
The first `variable` is a pointer, which is used to pass to `flag.BoolVar`. The second one is its value, which is whatever was set when the variable was declared.
They are Booleans, thus the default value is `false`, which is good enough for `flag.BoolVar`